emby_client.py

```diff
@@ -20,7 +20,7 @@
 AUTH_PASSWORD_KEY = "Pw"
 
 # query param constants
-MP3_STREAM = "stream.mp3?static=true"
```

Comment:
Good catch I have mp3 and FLAC in my library. However its 99% mp3 so I never hit nor tried this case.